engines/director/score.cpp

```diff
@@ -551,6 +551,7 @@ void Score::update() {
 }
 
 void Score::renderFrame(uint16 frameId, RenderMode mode) {
+	uint32 start = g_system->getMillis(false);
```

Comment:
Why are you skipping recording this value?

Reply:
The time is only used for the debug log, isn't the idea to only record the values which affect the control flow or playback?